Tasks/AzureCLIV2/task.json

```diff
@@ -0,0 +1,198 @@
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "connectedServiceNameARM",
+            "aliases": [
+                "azureSubscription"
+            ],
+            "type": "connectedService:AzureRM",
+            "label": "Azure temp subscription",
+            "required": true,
+            "helpMarkDown": "Select an Azure resource manager subscription for the deployment",
+            "properties": {
+                "EndpointFilterRule": "ScopeLevel != ManagementGroup"
+            }
+        },
+        {
+            "name": "scriptType",
+            "type": "pickList",
+            "label": "Script Type",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Type of script: Bat/Shell/PowerShell/PowerShell Core script. Select Shell/PowerShell Core script when running on Linux agent or Batch/PowerShell/PowerShell Core script when running on Windows agent. PowerShell Core script can run on cross-platform agents (Linux, macOS, or Windows).",
+            "options": {
+                "batch": "Batch script",
+                "bash": "Shell script",
+                "ps": "Powershell script",
+                "pscore": "Powershell Core script"
+            }
+        },
+        {
+            "name": "scriptLocation",
+            "type": "pickList",
+            "label": "Script Location",
+            "defaultValue": "scriptPath",
+            "required": true,
+            "helpMarkDown": "Path to script: File path or Inline script",
+            "options": {
+                "inlineScript": "Inline script",
+                "scriptPath": "Script path"
+            }
+        },
+        {
+            "name": "scriptPath",
+            "type": "filePath",
+            "label": "Script Path",
+            "defaultValue": "",
+            "required": true,
+            "visibleRule": "scriptLocation = scriptPath",
+            "helpMarkDown": "Fully qualified path of the script(.bat or .cmd or .ps1 when using Windows based agent else .sh or .ps1 when using linux based agent) or a path relative to the the default working directory"
+        },
+        {
+            "name": "inlineScript",
+            "type": "multiLine",
+            "label": "Inline Script",
+            "defaultValue": "",
+            "required": true,
+            "visibleRule": "scriptLocation = inlineScript",
+            "helpMarkDown": "You can write your scripts inline here. When using Windows agent, use batch or PowerShell or PowerShell Core scripting whereas use shell or PowerShell Core scripting when using Linux based agents. For batch files use the prefix \"call\" before every azure command. You can also pass predefined and custom variables to this script using arguments \n\n example for shell: \naz --version \naz account show \n\n example for batch:\ncall  az --version \ncall az account show",
```

Comment:
prefer PowerShell and PowerShell Core over batch when specifying these in order.

you want add an example for PS/PSCore?